Fixed a bug when there is spaces in file name (#21303)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/1_9_4.md

#### Integrations
##### CrowdStrike Falcon

> - Fixed an issue where the ***cs-falcon-rtr-remove-file*** command didn't work when there were spaces in the file name.

Reviewer: ```suggestion
- Fixed an issue where the ***cs-falcon-rtr-remove-file*** command failed when file name argument included spaces.
```
